src/rufus.c

```diff
@@ -2966,6 +2977,8 @@ static void PrintUsage(char* appname)
 	printf("     Start in GUI mode (disable the 'rufus.com' commandline hogger)\n");
 	printf("  -i PATH, --iso=PATH\n");
 	printf("     Select the ISO image pointed by PATH to be used on startup\n");
```

Comment:
This should read `IMAGE FILE` instead of `ISO image` to cater for both ISO and DD image files (although the `--iso=` could be changed to `--image=` or just add it to keep backward compatibility)

Reply:
I'm planning to do just that in 3.0.